tests/Test/Concerns/CreatesForum.php

```diff
@@ -108,23 +105,10 @@ protected function teardownApplication()
 
     protected function getFlarumConfig()
     {
-        $dbConfig = $this->configuration->getDatabaseConfiguration();
-
         return [
-            'debug'    => $this->configuration->isDebugMode(),
-            'database' => [
-                'driver'    => $dbConfig['driver'],
-                'host'      => $dbConfig['host'],
-                'database'  => $dbConfig['database'],
-                'username'  => $dbConfig['username'],
-                'password'  => $dbConfig['password'],
-                'charset'   => 'utf8mb4',
-                'collation' => 'utf8mb4_unicode_ci',
-                'prefix'    => $dbConfig['prefix'],
-                'port'      => $dbConfig['port'],
-                'strict'    => false
-            ],
-            'url'      => $this->configuration->getBaseUrl(),
+            'debug'    => true,
+            'database' => $this->getDatabaseConfiguration()->getConfig(),
+            'url'      => 'http://flarum.local',
```

Comment:
This value `url` can't be hardcoded. We should be able to override it from env at least, so that we properly test in CI. This is especially true when running browser tests (in the future). The same might be true for debug, eg by checking what information is returned from the API when running debug on/off.

Reply:
True, but soon irrelevant, when I follow up with my testing PR. So I'll ignore this here if you don't mind. ;)